AnkiDroid/src/main/java/com/ichi2/utils/ImportUtils.java

```diff
@@ -141,11 +141,11 @@ private static void sendShowImportFileDialogMsg(String path) {
     }
 
     public static boolean isCollectionPackage(String filename) {
-        return filename != null && (filename.toLowerCase().endsWith(".colpkg") || filename.equals("collection.apkg"));
+        return filename != null && (filename.toLowerCase().endsWith(".colpkg") || filename.toLowerCase().endsWith(".colpkg.zip") || (filename.equals("collection.apkg") || filename.equals("collection.apkg.zip")));
     }
 
     private static boolean isDeckPackage(String filename) {
-        return filename != null && filename.toLowerCase().endsWith(".apkg") && !filename.equals("collection.apkg");
+        return filename != null && (filename.toLowerCase().endsWith(".apkg") || filename.toLowerCase().endsWith(".apkg.zip")) && !filename.equals("collection.apkg");
```

Comment:
I think both of them should handle it if any do?

```suggestion
        return filename != null && (filename.toLowerCase().endsWith(".apkg") || filename.toLowerCase().endsWith(".apkg.zip")) && !(filename.equals("collection.apkg") || filename.equals("collection.apkg.zip"));
```

Reply:
Makes sens. I only clicked on "commit suggestion" in github, since your change seems to make change. I didn't check further. I'll take a look now